### ISSUE TITLE
error

### ISSUE THREAD

**Issue by bobtongue** [nZEDb/nZEDb#2422]
I just found this in my php error logs. I don't know what it means but there are a lot of them. Started happening after I restored db from backup. I was hoping someone could tell me what was causing it and how to fix it.

[23-Mar-2017 19:26:20 America/New_York] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42000]: Syntax error or access violation: 1305 PROCEDURE nzedb.delete_release does not exist in /var/www/myserver.com/htdocs/nZEDb/nzedb/Releases.php:635
Stack trace:
#0 /var/www/myserver.com/htdocs/nZEDb/nzedb/Releases.php(635): PDOStatement->execute()
#1 /var/www/myserver.com/htdocs/nZEDb/nzedb/ReleaseRemover.php(1096): nzedb\Releases->deleteSingle(Array, Object(nzedb\NZB), Object(nzedb\ReleaseImage))
#2 /var/www/myserver.com/htdocs/nZEDb/nzedb/ReleaseRemover.php(361): nzedb\ReleaseRemover->deleteReleases()
#3 /var/www/myserver.com/htdocs/nZEDb/nzedb/ReleaseRemover.php(263): nzedb\ReleaseRemover->removeGibberish()
#4 /var/www/myserver.com/htdocs/nZEDb/misc/testing/Release/removeCrapReleases.php(67): nzedb\ReleaseRemover->removeCrap(true, '4', 'gibberish', '')
#5 {main}
  thrown in /var/www/myserver.com/htdocs/nZEDb/nzedb/Releases.php on line 635

**Comment by Bart39** [comment]
Your missing the stored procedures from the schema (if your backup is pre these being added)

Manually add them starting from https://github.com/nZEDb/nZEDb/blob/0.x/resources/db/schema/mysql-ddl.sql#L1413

**Comment by bobtongue** [comment]
Thanks Bart. How do I go about entering those manually?

**Comment by bobtongue** [comment]
I just read that mysqldump does not backup stored procedures by default. Guess I have to change my backup command.  I really don'r know anything about these or how to get them back in the db.
@Bart39

**Comment by bobtongue** [comment]
I think I got it.
I copied the procedures section that you specified
I added 
USE nzedb;
DELIMITER $$
at the beginning
Ran mysql -u nzedb -p < storedprocedures.sql
and it inserted the procedures.
I checked in phpmyadmin and they are there.
I feel pretty proud of myself.
Changed my mysqldump command to include --routines so that it backs up the stored procedures
Hopefully all will be well now.
Thanks for the help @Bart39 .
